server/adapters/postgres.go

```diff
@@ -90,6 +94,9 @@ func (dsc *DataSourceConfig) Validate() error {
 	if dsc.Db == "" {
 		return errors.New("Datasource db is required parameter")
 	}
+	if strict && dsc.Schema == "" {
```

Comment:
Same. Schema isn't required parameter. It has default value (see Postgres/Redshift storage)

Reply:
Reverted